container/help.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+DOCKER="/usr/bin/docker"
+SELF=$1
+
+echo "OpenSCAP is an auditing tool that utilizes the Extensible Configuration
+Checklist Description Format (XCCDF). XCCDF is a standard way of expressing
+checklist content and defines security checklists.
+"
```

Comment:
I don't think this is the best description of OpenSCAP. Why it talks about XCCDF format only? Could it mention SCAP standard first? Do you think we could describe what OpenSCAP provides in terms of container scanning?

Reply:
I just put the text from RH container registry there: https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/rhel7/openscap

Reply:
@jan-cerny What should we put there? Do you have some better short description which can be used?

Reply:
@matusmarhefka OK, then we might want to update the description in Red Hat Registry as well.  I think the XCCDF support isn't the most exciting feature. It's not a big problem, I just think the description could be better.

Personally, I would mention there what is SCAP, what scan types the image provides maybe what are main use-cases.